Sources/OktaLogger/Analytics/MSAppCenterAnalytics.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Foundation
+import AppCenterAnalytics
+
+/**
+ AnalyticsProvider interface for different providers
+ eg: `AppCenter` implements this interface tracking functions
+ */
+
+@objc
+public protocol AppCenterAnalyticsProviderProtocol: AnyObject {
```

Comment:
why does this class needs to import  AppCenterAnalytics ? There should be default implementation that translates to a call made to AppCenterAnalytics

Reply:
we are injecting the `AppCenterAnalytics` as dependency, so we need the import on this class AppCenterAnalyticsProviderProtocol